src/routers/courseItem.ts

```diff
@@ -1,24 +1,59 @@
 import express from 'express';
+import { create, read, update, del } from '../operators/courseItemOperations';
 
 const cItemRouter = express.Router();
 
-cItemRouter.get('/', (req, res) => {
-	console.log('Get Course Item');
-	res.send('Get cItemRouter');
+cItemRouter.get('/:id', async (req, res) => {
+	try {
+		console.log('Get course Item');
+		const id = req.params.id;
+		const userID = req.header('userID');
+		if (id && userID) {
+			const courseItem = await read(id, userID);
+			console.log(courseItem);
+			res.send(courseItem);
+		} else {
+			throw 'ERROR - id undefined';
+		}
+	} catch (err) {
+		res.status(404).send(err);
```

Comment:
We don't want to send the err. We want to send "Not found" and no other information.